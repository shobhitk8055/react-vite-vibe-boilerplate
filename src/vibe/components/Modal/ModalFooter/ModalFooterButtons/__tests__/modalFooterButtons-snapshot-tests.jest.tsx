import React from "react";
import renderer from "react-test-renderer";
import ModalFooterButtons from "../ModalFooterButtons";

describe("ModalFooterButtons", () => {
  it("should render correctly", () => {
    const tree = renderer
      .create(<ModalFooterButtons primaryButtonText="Confirm" secondaryButtonText="Cancel" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
