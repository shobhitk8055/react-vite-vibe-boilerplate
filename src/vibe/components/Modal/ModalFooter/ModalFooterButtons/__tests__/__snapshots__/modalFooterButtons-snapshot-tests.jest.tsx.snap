// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooterButtons should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="container directionRow justifyEnd alignCenter"
    style={
      Object {
        "gap": "8px",
      }
    }
  >
    <button
      aria-busy={false}
      aria-disabled={false}
      className="button sizeMedium kindTertiary colorPrimary"
      data-testid="button"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      type="button"
    >
      Cancel
    </button>
    <button
      aria-busy={false}
      aria-disabled={false}
      className="button sizeMedium kindPrimary colorPrimary"
      data-testid="button"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      type="button"
    >
      Confirm
    </button>
  </div>
</div>
`;
